test/config.json

```diff
@@ -748,7 +748,7 @@
 			"Manual": false,
 			"Shard": "vtgate_schema_tracker",
 			"RetryMax": 1,
-			"Tags": []
+			"Tags": ["upgrade_downgrade_query_serving"]
```

Comment:
Is it expected that there will be more tags of this kind?

Reply:
I am expecting to have at least two more, one for backups and one for cluster management